yarn.lock

```diff
@@ -0,0 +1,3544 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
The project uses npm, try to not add additional lockfiles to the repo. One of em could get out of date, atleast until npm7 is released.